at_client/lib/src/util/at_client_util.dart

```diff
@@ -135,6 +134,13 @@ class AtClientUtil {
     }
     return atKey.key!;
   }
+
+  static String getVersion() {
+    var applicationConfiguration = ApplicationConfiguration('pubspec.yaml');
```

Comment:
This is dependent on what the "current working directory" is ...

Suggest you use this approach instead: https://stackoverflow.com/a/72486732

Reply:
Explored the above, the rootBundle is a flutter dependent package. Hence unable to use it in a Dart package.